file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,589 @@
+
+      "! <p class="shorttext">Code</p>
+      "! Code
+      "! $required
+      "! $maxLength 30
+      code        TYPE ty_code_value,
+
+      "! <p class="shorttext">Description</p>
+      "! Description
+      "! $required
+      "! $maxLength 60
+      description TYPE c LENGTH 60,
+    END OF ty_gs_code_list_value_descr.
+  TYPES: ty_gt_code_list_value_descr TYPE STANDARD TABLE OF ty_gs_code_list_value_descr WITH KEY language code.
+
+  TYPES:
+    "! <p class="shorttext">Code Value</p>
+    "! Code value
+    BEGIN OF ty_gs_code_list_value,
+      "! <p class="shorttext">Code</p>
+      "! Code
+      "! $required
+      "! $maxLength 30
+      code        TYPE ty_code_value,
+
+      "! <p class="shorttext">Disabled</p>
+      "! Disabled
+      is_disabled TYPE abap_bool,
+    END OF ty_gs_code_list_value.
+  TYPES: ty_gt_code_list_value TYPE STANDARD TABLE OF ty_gs_code_list_value WITH KEY code.
+
+  TYPES:
+    "! <p class="shorttext">Value Help Key Mapping Details</p>
+    "! Value help key mapping details
+    BEGIN OF ty_gs_value_help_key_mapping,
+      "! <p class="shorttext">Component Identifier</p>
+      "! Component identifier
+      "! $maxLength 5
+      component_identifier      TYPE ty_component_identifier,
+
+      "! <p class="shorttext">Value Help Field Name</p>
+      "! Value help field name
+      "! $required
+      "! $maxLength 30
+      value_help_field_name     TYPE ty_field_name,
+
+      "! <p class="shorttext">Dimension View Field Name</p>
+      "! Dimension view field name
+      "! $maxLength 30
+      dimension_view_field_name TYPE ty_field_name,
+    END OF ty_gs_value_help_key_mapping.
+  TYPES ty_gt_value_help_key_mapping TYPE STANDARD TABLE OF ty_gs_value_help_key_mapping WITH KEY component_identifier.
+
+  TYPES:
+    "! <p class="shorttext">Value Help Binding Details</p>
+    "! Value help binding details
+    BEGIN OF ty_gs_value_help_binding,
+      "! <p class="shorttext">Extended CDS View Name</p>
+      "! Extended CDS view name
+      "! $required
+      "! $maxLength 30
+      cds_view_name                TYPE  zif_aff_types_v1=>ty_object_name_30,
+
+      "! <p class="shorttext">Extended CDS View Field Name</p>
+      "! Extended CDS view field name
+      "! $required
+      "! $maxLength 30
+      extended_cds_view_field_name TYPE ty_field_name,
+
+      "! <p class="shorttext">Value Help Field Name</p>
+      "! Value help field name
+      "! $required
+      "! $maxLength 30
+      value_help_field_name        TYPE  ty_field_name,
+    END OF ty_gs_value_help_binding.
+  TYPES ty_gt_value_help_binding TYPE STANDARD TABLE OF ty_gs_value_help_binding
+                                 WITH KEY cds_view_name value_help_field_name.
+
+  TYPES:
+    "! <p class="shorttext">Data Protection and Privacy</p>
+    "! Data protection and privacy
+    BEGIN OF ty_gs_dpp_properties,
+      "! <p class="shorttext">Data Subject</p>
+      "! Data subject
+      "! $maxLength 10
```

Comment:
Like above, maxLength 10 is wrong, here, since it is the maxLength of the JSON value (notApplicable) and not the ABAP Value (" ")

Reply:
All occurrences of $maxLength were removed...